src/protocols/secure_channel/CASEServer.cpp

```diff
@@ -181,4 +179,30 @@ void CASEServer::OnSessionEstablished(const SessionHandle & session)
                     ChipLogValueScopedNodeId(session->GetPeer()));
     PrepareForSessionEstablishment(session->GetPeer());
 }
+
+void CASEServer::SendBusyStatusReport(Messaging::ExchangeContext * ec)
+{
+    using namespace Protocols::SecureChannel;
+
+    ChipLogProgress(SecureChannel, "Already in the middle of CASE handshake, sending busy status report");
+
+    // A successful CASE handshake can take several seconds and some may timeout (30 seconds).
+    // TODO: Come up with better estimate: https://github.com/project-chip/connectedhomeip/issues/28288
+    // For now, setting minimum wait time to 5 seconds.
+    uint16_t minimumWaitTime                            = 5 * 1000; // milliseconds
+    constexpr uint8_t kBusyStatusReportProtocolDataSize = 2;        // 16-bits
```

Comment:
Any reason not to do:
```suggestion
    constexpr uint8_t kBusyStatusReportProtocolDataSize = sizeof(minimumWaitTime);        // 16-bits
```
?